lib/CMakeLists.txt

```diff
@@ -82,6 +82,11 @@ if (FL_LIBRARIES_USE_CUDA)
     $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/..> # TODO(jacobkahn): fix me
     )
 
+  # TODO For CUDA version 11 CUB doesn't have to be built 
+  if (CUDA_VERSION VERSION_GREATER "11.0" OR CUDA_VERSION VERSION_EQUAL "11.0")
+    add_compile_definitions(THRUST_IGNORE_CUB_VERSION_CHECK)
```

Comment:
Let's use `target_compile_definitions` on the `lib` target. We can make the modification `PUBLIC` so it works with libraries that link to `lib`.

Reply:
done